Formula/sourcedocs.rb

```diff
@@ -0,0 +1,21 @@
+class Sourcedocs < Formula
+  last_release = "0.5.1"
+
+  desc "Generate Markdown files from inline source code documentation"
+  homepage "https://github.com/eneko/SourceDocs"
+  url "https://github.com/eneko/SourceDocs/archive/#{last_release}.tar.gz"
+  sha256 "3c2e2de695d49dbdd5acb49f8876042bdc97e8d6b95584d3ef6b592b8f10affc"
+
+  depends_on :xcode => ["9.3", :build]
+
+  def install
+    ohai "Building SourceDocs"
```

Comment:
Please remove `ohai`, this generally should not be used. If the user really wants to know what is going on, the user should pass `--verbose`.